### ISSUE TITLE
Lack of diversity in the 1ake example prediction by DiG

### ISSUE THREAD

**Issue by PKUfjh** [microsoft/Graphormer#183]
I tried to use the newly open-sourced DiG code to make 1ake example prediction for 100 samples, but all conformations are mainly around 1ake experimental structure, deviate away from 1ake structure. The result is shown in the figure.
![overlay_tmscore_1ake](https://github.com/microsoft/Graphormer/assets/32134282/c14131de-16d5-4ae4-96da-a3c70d36c3b8)
 To be more specific, the command I used for prediction is
```bash
PDBID="1ake"
CKPT_PATH=./checkpoints/checkpoint-520k.pth
FEATURE_PATH=./dataset/${PDBID}.pkl
FASTA_PATH=./dataset/${PDBID}.fasta
OUTDIR=./output/${PDBID}/
mkdir -p ${OUTDIR}
python run_inference.py -c ${CKPT_PATH} -i ${FEATURE_PATH}\
 -s ${FASTA_PATH} -o ${PDBID} --output-prefix ${OUTDIR}\
 -n 100 --use-gpu --use-tqdm
```

**Comment by zhengsx** [comment]
the energy of 4ake is much higher than 1ake, thus much more samples are needed to observe structures around 4ake.

**Comment by sai-advaith** [comment]
Hi,

How were you able to obtain the representations for 4ake? I did not find it in the dataset.

**Comment by PKUfjh** [comment]
No, since 1ake and 4ake has the same sequence, the input is the same with 1ake for 4ake.发自我的iPhone------------------ Original ------------------From: Advaith  ***@***.***>Date: Tue,Apr 23,2024 6:59 PMTo: microsoft/Graphormer ***@***.***>Cc: PKUfjh ***@***.***>, Author ***@***.***>Subject: Re: [microsoft/Graphormer] Lack of diversity in the 1ake exampleprediction by DiG (Issue #183)
Hi,
How were you able to obtain the representations for 4ake? I did not find it in the dataset.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/microsoft/Graphormer/issues/183#issuecomment-2072005338",
"url": "https://github.com/microsoft/Graphormer/issues/183#issuecomment-2072005338",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by sai-advaith** [comment]
Thanks for the response.

When you get the pdb output from the code, were you able to visualize the sidechains too? We get a pdb with backbones only.

**Comment by PKUfjh** [comment]
> Thanks for the response.
> 
> When you get the pdb output generated from the code, were you able to visualize the sidechains too? We get a pdb with backbones only.

No, I think the output is only the protein backbones.

**Comment by Paulie-ai** [comment]
Hi, I used AF2 repr from evoformer directly for pdbid 1ake and not used dataset provided, i find the sample is almost same as ground truth 1ake structure, why? Is my pipeline is differ with paper?  single and pair repe is l, 384 and l, l,128 as provided shape. Thanks

**Comment by zxzhang0507** [comment]
I have the same problem. I generated 3000 conformations for the 1AKE system using the officially provided .pkl file and the checkpoint-520k.pth model parameters, and these structures are almost identical. Someone replied earlier that 4ake energy is higher than 1ake, I used AF2 to generate a .pkl file for the 4ake system and generated 10,000 conformations, also still essentially no conformational shift.

**Comment by LifeWorks** [comment]
@PKUfjh Hi I assume you downloaded the datasets and checkpoints successfully, the token expired in May because of Microsoft policy. I wonder would you mind share what you have downloaded? Thanks very much!

**Comment by hypefolder** [comment]
Has there been any update to this @zhengsx ? How many conformations did you need to generate for the structure shown in the paper that is similar to 4ake?

Pretty odd that the model weights and data suddenly disappear after an example in the paper fails to be replicated.

> the energy of 4ake is much higher than 1ake, thus much more samples are needed to observe structures around 4ake.

**Comment by jeevster** [comment]
I am also curious about this @zhengsx. How many conformations need to be sampled before you get a structure similar to 4ake?
